### PR TITLE
PerconaXtraDB doc

### PR DESCRIPTION
**Pull request by shudipta**

### CODE REVIEW

**Comment by tamalsaha** [comment]
- [ ] Custom RBAC doc not written, broken link. /docs/guides/percona-xtradb/custom-rbac/using-custom-rbac.md . https://github.com/kubedb/docs/tree/master/docs/guides/mysql/custom-rbac

File: docs/guides/percona-xtradb/clustering/percona-xtradb-cluster.md [hunk 1]
```diff
@@ -0,0 +1,610 @@
...
+
+# KubeDB - Percona XtraDB Cluster
+
+This tutorial will show you how to use KubeDB to provision a Percona XtraDB Cluster.
+
+## Before You Begin
+
+Before proceeding:
+
+- Read [Percona XtraDB Cluster](/docs/guides/percona-xtradb/clustering/overview.md).
```
Comment (tamalsaha) [inline docs/guides/percona-xtradb/clustering/percona-xtradb-cluster.md]: No such doc exists.

File: docs/concepts/databases/percona-xtradb.md [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+
+#### .spec.podTemplate.spec.serviceAccountName
+
+ `serviceAccountName` is an optional field supported by KubeDB Operator (version 0.13.0 and higher) that can be used to specify a custom service account to fine tune role based access control.
+
+ If this field is left empty, the KubeDB operator will create a service account name matching PerconaXtraDB object name. Role and RoleBinding that provide necessary access permissions will also be generated automatically for this service account.
+
+ If a service account name is given, but there's no existing service account by that name, the KubeDB operator will create one, and Role and RoleBinding that provide necessary access permissions will also be generated for this service account.
+
+ If a service account name is given, and there's an existing service account by that name, the KubeDB operator will use that existing service account. Since this service account is not managed by KubeDB, users are responsible for providing necessary access permissions manually. Follow the guide [here](/docs/guides/percona-xtradb/custom-rbac/using-custom-rbac.md) to grant necessary permissions in this scenario.
```
Comment (tamalsaha) [inline docs/concepts/databases/percona-xtradb.md]: No such doc exists

File: docs/concepts/databases/percona-xtradb.md [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+  name: init-demo-px
+spec:
+  version: 5.7
+  init:
+    scriptSource:
+      configMap:
+        name: my-init-script
+```
+
+In the above example, KubeDB operator will launch a Job to execute all js script of `my-init-script` in alphabetical order once StatefulSet Pods are running. For more details tutorial on how to initialize from script, please visit [here](/docs/guides/percona-xtradb/using-script.md).
```
Comment (tamalsaha) [inline docs/concepts/databases/percona-xtradb.md]: No such doc exists.

File: docs/guides/percona-xtradb/quickstart/quickstart.md [hunk 4]
```diff
@@ -0,0 +1,492 @@
...
+---
+
+> New to KubeDB? Please start [here](/docs/concepts/README.md).
+
+# PerconaXtraDB QuickStart
+
+This tutorial will show you how to use KubeDB to run a PerconaXtraDB database.
+
+<p align="center">
+  <img alt="lifecycle"  src="/docs/images/mysql/mysql-lifecycle.png">
```
Comment (tamalsaha) [inline docs/guides/percona-xtradb/quickstart/quickstart.md]: Why use the MySQL lifecycle image?

File: docs/guides/percona-xtradb/configuration/using-custom-config.md [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+  namespace/demo created
+  ```
+
+> Note: YAML files used in this tutorial are stored in [docs/examples/percona-xtradb](https://github.com/kubedb/docs/tree/{{< param "info.version" >}}/docs/examples/percona-xtradb) folder in GitHub repository [kubedb/docs](https://github.com/kubedb/docs).
+
+## Overview
+
+PerconaXtraDB allows to configure database via configuration file. The default configuration for PerconaXtraDB can be found in `/etc/my.cnf` file. When PerconaXtraDB starts, it looks for custom configuration files.
+
+- For standalone server, it looks into `/etc/my.cnf.d/`and `/etc/percona-server.conf.d/` directories. Here, KubeDB uses the later one `/etc/percona-server.conf.d/` for custom configurations. If configuration file exists, the `mysqld` will use combined startup setting from both `/etc/my.cnf` and `*.cnf` files in `/etc/percona-server.conf.d/` directory. This custom configuration will overwrite the existing default one.
```
Comment (tamalsaha) [inline docs/guides/percona-xtradb/configuration/using-custom-config.md]: How do we know which folder to look into?

File: docs/concepts/databases/percona-xtradb.md (hunk 3 above)
Comment (shudipta) [reply docs/concepts/databases/percona-xtradb.md]: resolved

File: docs/concepts/databases/percona-xtradb.md (hunk 2 above)
Comment (shudipta) [reply docs/concepts/databases/percona-xtradb.md]: resolved

File: docs/guides/percona-xtradb/clustering/percona-xtradb-cluster.md (hunk 1 above)
Comment (shudipta) [reply docs/guides/percona-xtradb/clustering/percona-xtradb-cluster.md]: resolved

File: docs/guides/percona-xtradb/quickstart/quickstart.md (hunk 4 above)
Comment (shudipta) [reply docs/guides/percona-xtradb/quickstart/quickstart.md]: resolved

**Pull request merged**